docs/reference/redirects.asciidoc

```diff
@@ -1811,10 +1811,39 @@ even if you opt to perform a full-cluster restart instead of a rolling upgrade.
 For more information about upgrading, see 
 {stack-ref}/upgrading-elastic-stack.html[Upgrading to Elastic {version}].
 
-role="exclude",id="rolling-upgrade"]
+[role="exclude",id="rolling-upgrades"]
 === Rolling upgrade
 
 When upgrading to {es} 8.0 and later, you must first upgrade to {prev-major-last}
 whether you opt to perform a rolling upgrade (upgrade one node at a time) or a full-cluster restart upgrade.
 For more information about upgrading, see 
 {stack-ref}/upgrading-elastic-stack.html[Upgrading to Elastic {version}].
+
+[role="exclude",id="reindex-upgrade"]
+=== Reindex before upgrading
+
+//tag::upgrade-reindex[]
+Before upgrading to {es} 8.0 and later, you must reindex any indices created in
+a 6.x version. We recommend you use the Upgrade Assistant to guide you
+through this process.
+
+For more information about upgrading, see 
```

Comment:
In Kibana, the writers changed this language to "refer to" rather than "see" as more inclusive language for nonsighted users. I've tried to start doing that in newer PRs where there's a reference link. Just raising for awareness if you want to update the references in this PR.